compiler/test/suites/imports.re

```diff
@@ -14,175 +14,313 @@ describe("imports", ({test, testSkip}) => {
   /* import * tests */
   assertRun(
     "import_all",
-    "import * from \"exportStar\"; {print(x); print(y(4)); print(z)}",
+    {|
+      import * from "exportStar"
+      {
+        print(x)
+        print(y(4))
+        print(z)
+      }
+    |},
     "5\n4\nfoo\n",
```

Comment:
What about these "expected" outputs? It's weird that we need to use `\n` in the string, isn't it?

Reply:
I agree.